coffee-backend/src/main/resources/credentials/secrets.properties

```diff
@@ -0,0 +1,8 @@
+postgres.username = username
+postgres.databasename = databasename
+postgres.password = password
+postgres.url = jdbc:postgresql://example.com:5432
+
+# Example VAPID keys - Must be valid for bean instantiation
+vapid.key.public = BMCygTJUtzWRPliOM5BYT150OvjMTq0UHtYmrZWllT73ftq4eYSRCJnDWAd_aiSHP5Ou78j8AbVqOG3k5TzRXBw
+vapid.key.private = BokoKjBF7016VHZ6IO_MAYTbGt8RY6RU0dtWlid6tL4
```

Comment:
Uhmm?

Reply:
Just a randomly generated point on an elliptic curve, not the public/private keys we actually use in production. Will create our own config parser, so that we can get rid of this checked in file and use ENV variables on CI builds instead